test/unit/sql/insert.test.js

```diff
@@ -93,6 +93,7 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
           bind: {
             ibmi: { sequelize_1: '2015-01-20 00:00:00.000' },
             sqlite: { sequelize_1: '2015-01-20 00:00:00.000 +00:00' },
+            // TODO!: for this test and the test below check why the result is different
```

Comment:
And here

Reply:
This one has been resolved in the main PR by https://github.com/sequelize/sequelize/pull/14505/commits/a71035e67c72ff62a70d46fe192f8ccd0157319d